Packs/RansomwareResponsePackForICSSystemsUsingCortexXSOARAndXDR/layoutscontainer/layoutscontainer-Claroty_Security_Incident_layout_(custom).json

```diff
@@ -0,0 +1,709 @@
+        "sectionItemType": "field",
+        "startCol": 0
+       },
+       {
+        "endCol": 2,
+        "fieldId": "lastactivity",
+        "height": 22,
+        "id": "9316a4e0-967b-11ea-931b-7513e8008c32",
+        "index": 4,
+        "sectionItemType": "field",
+        "startCol": 0
+       },
+       {
+        "dropEffect": "move",
+        "endCol": 2,
+        "fieldId": "vendorproduct",
+        "height": 22,
+        "id": "c24b4cc0-967b-11ea-931b-7513e8008c32",
+        "index": 5,
+        "listId": "4ugqr3mvsi-627ceec0-967b-11ea-931b-7513e8008c32",
+        "sectionItemType": "field",
+        "startCol": 0
+       },
+       {
+        "endCol": 2,
+        "fieldId": "sitename",
+        "height": 44,
+        "id": "baf93720-967b-11ea-931b-7513e8008c32",
+        "index": 6,
+        "sectionItemType": "field",
+        "startCol": 0
+       },
+       {
+        "endCol": 2,
+        "fieldId": "virtualzone",
+        "height": 44,
+        "id": "c594f980-967b-11ea-931b-7513e8008c32",
+        "index": 7,
+        "sectionItemType": "field",
+        "startCol": 0
+       },
+       {
+        "endCol": 2,
+        "fieldId": "name",
+        "height": 22,
+        "id": "07113220-967c-11ea-931b-7513e8008c32",
+        "index": 8,
+        "sectionItemType": "field",
+        "startCol": 0
+       }
+      ],
+      "maxW": 3,
+      "minH": 1,
+      "minW": 1,
+      "moved": false,
+      "name": "Detailed Information",
+      "static": false,
+      "w": 3,
+      "x": 0,
+      "y": 4
+     }
+    ],
+    "type": "custom"
+   },
+   {
+    "hidden": false,
+    "id": "yrgjffhitm",
+    "name": "CVE and Impacted ICS",
+    "sections": [
+     {
+      "displayType": "CARD",
+      "h": 6,
+      "hideItemTitleOnlyOne": false,
+      "hideName": false,
+      "i": "yrgjffhitm-e9ee03d0-9810-11ea-9f9b-e7a8a52b0adc",
+      "items": [
+       {
+        "dropEffect": "move",
+        "endCol": 1,
+        "fieldId": "impactedicssystems",
+        "height": 106,
+        "id": "bc410600-9813-11ea-b3c4-2f3eead3d4a8",
+        "index": 0,
+        "listId": "yrgjffhitm-e9ee03d0-9810-11ea-9f9b-e7a8a52b0adc",
+        "sectionItemType": "field",
+        "startCol": 0
+       },
+       {
+        "dropEffect": "move",
+        "endCol": 2,
+        "fieldId": "vulnerability",
+        "height": 106,
+        "id": "fedfcd00-9810-11ea-9f9b-e7a8a52b0adc",
+        "index": 0,
+        "listId": "yrgjffhitm-e9ee03d0-9810-11ea-9f9b-e7a8a52b0adc",
+        "sectionItemType": "field",
+        "startCol": 1
+       }
+      ],
+      "maxW": 3,
+      "minH": 1,
+      "minW": 1,
+      "moved": false,
+      "name": "Impacted ICS System and CVE Information",
+      "static": false,
+      "w": 3,
+      "x": 0,
+      "y": 0
+     }
+    ],
+    "type": "custom"
+   }
+  ]
+ },
+ "group": "incident",
+ "id": "96790ed3-c42f-46a9-800b-2d0629b511f0",
+ "name": "Claroty Security Incident layout (custom)",
```

Comment:
Please provide a better name other than custom, and change the id to match the name.
You should also add a new incident type and a classifier for this incident type / layout.

Reply:
Hi @DeanArbel You may replace it with "ICS case details" or "ICS (Industrial control system) system details"